internal/testgen/gen_contains.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"strings"
+	"text/template"
+
+	"github.com/Antonboom/testifylint/internal/checkers"
+)
+
+type ContainsTestsGenerator struct{}
+
+func (ContainsTestsGenerator) Checker() checkers.Checker {
+	return checkers.NewContains()
+}
+
+func (g ContainsTestsGenerator) TemplateData() any {
+	checker := g.Checker().Name()
+
+	return struct {
+		CheckerName       CheckerName
+		InvalidAssertions []Assertion
+		ValidAssertions   []Assertion
+		IgnoredAssertions []Assertion
+	}{
+		CheckerName: CheckerName(checker),
+		InvalidAssertions: []Assertion{
+			{
+				Fn:            "True",
+				Argsf:         `strings.Contains(s, "abc123")`,
+				ReportMsgf:    checker + ": use %s.%s",
+				ProposedFn:    "Contains",
+				ProposedArgsf: `s, "abc123"`,
```

Comment:
\+ `[]byte` case 

```go
assert.True(t, strings.Contains(string(b), "abc123"))
```
->
```go
assert.Contains(t, string(b), "abc123")
````

while byte-contains doesn't support https://github.com/Antonboom/testifylint/issues/54#issuecomment-1933841930

Reply:
Good catch is used a lot

https://github.com/search?q=language%3Ago+%22assert.Contains%28t%2C+string%28%22&type=code

Reply:
Blocked by testify unfortunately, Anton opened #154 to support it later, good catch

Reply:
@mmorel-35 

**Blocked by testify unfortunately**

you should ignore it, and not add to valid examples :)

but anyway I will fix it in review-commit, later